ceph/ceph_admin/host.py

```diff
@@ -105,6 +107,15 @@ def add(self, config):
         # Add host
         self.shell(args=cmd)
 
+        if _labels:
```

Comment:
I think it would be good, if this check moved to last section/validation, since there are already few checks like host, label and address validation. so it would be right place for `_admin` validation.

Reply:
Ack, I made these changes